reference/tax_customers.v3.yml

```
@@ -0,0 +1,189 @@
+                  type: object
+                  properties:
+                    data:
+                      type: array
+                      items:
+                        $ref: "#/components/schemas/TaxCustomer"
+                    meta:
+                      $ref: "#/components/schemas/MetaOpen"
+          "400":
+            description: Request parameters invalid
+      put:
+        tags:
+          - Tax Customers
+        summary: Update Tax Customers
+        description: |-
+          Update the tax data associated with one or more customers.
+          This operation will be additive to any values already associated with the customer, overwriting any existing values.
+        operationId: updateTaxCustomer
+        parameters:
+          - $ref: '#/components/parameters/ContentType'
+        requestBody:
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/TaxCustomer"
+              examples:
+                Example:
+                  value:
+                    - customer_id: 157
+                      tax_properties:
+                        A-123456789: "26"
+                        B-6731789: "200"
+          required: true
+        responses:
+          "200":
+            description: OK
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    data:
+                      type: array
+                      items:
+                        $ref: "#/components/schemas/TaxCustomer"
+                    meta:
+                      $ref: "#/components/schemas/MetaOpen"
+          "422":
+            description: The request body does not meet specifications.
+      delete:
+        tags:
+          - Tax Customers
+        summary: Delete Tax Customers
+        description: Delete tax data that is associated with one or more customers.
+        operationId: deleteTaxCustomer
+        parameters:
+          - $ref: "#/components/parameters/customer_idin"
+        responses:
+          "204":
+            description: No Content
+          "400":
+            description: Request parameters invalid
+components:
+  parameters:
+    Accept:
+      name: Accept
+      in: header
+      required: true
+      description: 'The [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) of the response body.'
+      schema:
+        type: string
+        default: 'application/json'
+    ContentType:
+      name: Content-Type
+      in: header
+      required: true
+      description: 'The [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) of the request body.'
+      schema:
+        type: string
+        default: 'application/json'
+    customer_idin:
+      name: customer_id:in
+      in: query
+      required: false
+      description: |-
```

```suggestion
      description: ID of customer. To target multiple customers, provide a comma-separated list of IDs such as `12,34,56`.
```

I think we can merge this? The two sentences are not too long for them to be placed as a separate lines.